src/Servers/Kestrel/test/FunctionalTests/ResponseTests.cs

```diff
@@ -812,6 +812,7 @@ await using (var server = new TestServer(App, testContext, listenOptions))
 
         [Fact]
         [CollectDump]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/26306")]
```

Comment:
@BrennanConroy @jkotalik @JunTaoLuo I couldn't see an easy way to quarantine this test just in the Libuv run. Is it worth creating a Libuv-specific subclass❔

Reply:
Is this failing only for libuv? If so, quarantining is fine I think.

Reply:
Yeah, it's only failing for Libuv from what I can see and it's only failed once since the test was added. Might be related to the newer dotnet/runtime versions introduced in this PR or just pure randomness.